web/static/js/components/user_list_item.jsx

```diff
@@ -8,13 +8,22 @@ function UserListItem(props) {
   return (
     <li className={`item ${styles.wrapper}`}>
       <div className="ui center aligned grid">
-        <div className="ui row">{ userName }</div>
-        <i className="huge user icon" />
+         {IconTag(props.user)}
+         <div className="ui row">{ userName }</div>
       </div>
     </li>
   )
 }
 
+const IconTag = function(user) {
+  if (user.picture) {
+    return <img className={`${styles.picture}`} src={user.picture} />
```

Comment:
no need to interpolate here for providing the `className` prop. you could just reference `styles.picture` outright